robottelo/cli/factory.py

```diff
@@ -1711,10 +1712,10 @@ def setup_org_for_a_custom_repo(options=None):
     if result.return_code != 0:
         raise CLIFactoryError('Failed to synchronize repository')
     # Create CV if needed and associate repo with it
-    cv_id = options.get(
-        'content-view-id',
-        make_content_view({u'organization-id': org_id})['id']
-    )
+    if options.get('content-view-id') is None:
```

Comment:
It doesn't work the way you expect it to work. `make_content_view({u'organization-id': org_id})['id']` will be ran, received `id` will be set to `default=` argument and then `options.get('content-view-id', default=*id*)` will be executed. As a result content view will always be created, even if `'content-view-id'` is not None. So it's not an option to use `get()` in such cases.